contracts/4/BridgedTokenOWL.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity ^0.4.24;
+
+import "../../vendor/tokenbridge-contracts/contracts/PermittableToken.sol";
+
+/// @dev A contract to implement OWL tokens on a bridged chain (e.g. XDai).
+/// Implements "burning" by allocating the amount to a dedicated receiver,
+/// which can transfer it to the native chain (e.g. Mainnet) and complete
+/// the burn there.
+contract BridgedTokenOwl is PermittableToken {
+    address public burnReceiver;
+
+    event Burnt(address indexed from, address indexed user, uint256 amount);
```

Comment:
Is this some standard event?

Reply:
It's the same we use on mainnet: https://github.com/gnosis/owl-token/blob/602477d11278e66c83cfe275cce71eab1320175d/contracts/5/TokenOWL.sol#L25

Reply:
It feels weird that because of that the 2 burn methods emit different events

Reply:
Why would they emit different events if they are declared the same?

Reply:
In this inheritance hierarchy, there are two events. 

1. The BurnableToken's `event Burn(address indexed burner, uint256 value);`
2. The BridgedTokenOwl's `event Burnt(address indexed from, address indexed user, uint256 amount);`

I agree that there name similarity is confusing, but so is their burn function names (burn vs. burnOWL) as well. I chose to use the very same event name and type as we use on mainnet so that our tooling that depends on certain events can continue to exist without changes.